go-controller/pkg/util/multi_network_test.go

```diff
@@ -270,17 +270,16 @@ func TestParseNetconf(t *testing.T) {
                 "vlanID": 10,
                 "netAttachDefName": "ns1/nad1"
               }
-            ],
+            ]
     }
 `,
 			expectedNetConf: &ovncnitypes.NetConf{
 				Topology: "localnet",
 				NADName:  "ns1/nad1",
 				MTU:      1400,
 				VLANID:   10,
-				NetConf:  cnitypes.NetConf{Name: "tenantred", Type: "ovn-k8s-cni-overlay"},
+				NetConf:  cnitypes.NetConf{Name: "tenantred", CNIVersion: "1.0.0", Type: "ovn-k8s-cni-overlay"},
```

Comment:
nit: maybe consider making 1.0.0 a constant

Reply:
since we don't easily reuse it, i think it is fine and better to be explicit here

Reply:
all good, it's a nit.